### PR TITLE
Added Sidebar, About-us section and Contributors page

### PR DESCRIPTION
**Pull request by rt1301**
Feature: I have added Sidebar, About us section and contributors page to the main site
Screenshots of Change:
Screenshot from 2021-01-22 23-26-34
<br>
Screenshot from 2021-01-22 23-16-21
<br>
Screenshot from 2021-01-22 23-16-41

Tagging @TanviKumar and @Jayashrri for review. Thanks!

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,21 +1,31 @@
...
+import AboutUs from "./components/AboutUs/AboutUs";
+import Contributors from "./components/Contributors/Contributors";
 import {
   BrowserRouter as Router,
   Route,
   Switch,
   useRouteMatch,
   useParams,
 } from "react-router-dom";
+import SideBar from "./components/SideBar/SideBar";
```
Comment (Jayashrri) [inline src/App.js]: Shift this import line up with the rest of the component imports

File: src/components/DashBoard/DashBoard.jsx [hunk 2]
```diff
@@ -5,6 +5,7 @@ import Grid from "@material-ui/core/Grid";
 import { Container } from "@material-ui/core";
 import FilterTags from "../FilterTags/FilterTags";
 import Axios from "axios";
+import Header from "../Header/Header";
```
Comment (Jayashrri) [inline src/components/DashBoard/DashBoard.jsx]: Put all the component imports together

**Comment by rt1301** [comment]
@Jayashrri I have made the above changes, please review it once. Thanks!

**Comment by rt1301** [comment]
@Jayashrri I have resolved the merge-conflicts, please have a look. Thanks!

**Review by Jayashrri — APPROVED** [review]

**Pull request merged**